docs/developers_guide/organization.rst

```diff
@@ -1105,6 +1105,32 @@ attributes.  Another reason might be to set these attributes using an algorithm
 When overriding ``constrain_resources``, it is important to also call the base
 class' version of the method with ``super().constrain_resources()``.
 
+The names of the resources are related to the
+`Slurm <https://slurm.schedmd.com/srun.html>`_ naming conventions:
+
+``ntasks``
+    The target number of MPI tasks that a step will use if the resources are
+    available.
+
+``min_tasks``
+    The minimum number of MPI tasks for a step.  If too few resources are
+    available, the step will not run.
+
+``cpus_per_task``
```

Comment:
For straight MPI applications without OpenMP, this will always be 1, right?  If so, can you add a statement to that effect?

Reply:
I added this.